src/app/space/create/deployments/apps/deployment-card.component.spec.ts

```diff
@@ -33,6 +33,11 @@ import { Environment } from '../models/environment';
 import { DeploymentsService } from '../services/deployments.service';
 import { DeploymentCardComponent } from './deployment-card.component';
 
+const ICON_OK = 'pficon-ok';
+const ICON_WARN = 'pficon-warning-triangle-o';
+const ICON_ERR = 'pficon-error-circle-o';
+const MSG_OK = 'Everything is ok.';
```

Comment:
Should these be duplicates or can you just export the values from the component and reference them here?